README.md

```diff
@@ -1,5 +1,40 @@
-# Node-RED Quantum
-Quantum computing functionality for Node-RED.
+# node-red-quantum
+
+[![platform](https://img.shields.io/badge/platform-Node--RED-red)](https://nodered.org)
+![NPM version]()
+![NPM]()
+
+**Node-RED Quantum** provides a set of nodes to build and run quantum computing algorithms within Node-RED. \
+This module is a user-friendly library that is suitable to new quantum computing users thanks to its extensive documentation. \
+It was designed to facilitate the integration of quantum algorithms within classical programs and is fully scalable since all the elemental quantum operations are included. 
```

Comment:
```suggestion
**Node-RED Quantum** provides a set of nodes to build and run quantum computing algorithms within Node-RED.

This module is a user-friendly library that is suitable to new quantum computing users thanks to its extensive documentation. It was designed to facilitate the integration of quantum algorithms within classical programs and is fully scalable since all the elemental quantum operations are included. 
```
I think this separation of sections reads much better once the markdown is parsed, since it starts with a clearly separate one-line introduction, and then goes on to a more extensiveness description.